casper/src/main/scala/coop/rchain/casper/util/rholang/RegistrySigGen.scala

```diff
@@ -73,6 +74,7 @@ final case class Derivation(
     | 3, 5, registry  6. value = ${pprint(toSign)}
     | 6,    protobuf  7. toSign = ${Hex(toSign.toByteArray)}
     | 7, 1, ed25519   8. sig = ${result.sig}
+    |                 9. uri = ${uri}
```

Comment:
The URI is derived from the public key, so the justification for this step should be something like `4,    registry` where "registry" is short for "the rules in the RChain registry, i.e. Blakeb256 hash and CRC14".

Reply:
Now I understand what the information on the left is. I'll fix this.